libs/platform/src/lib/components/action-bar/action-bar.component.ts

```diff
@@ -28,7 +31,7 @@ export class ActionBarComponent extends BaseComponent implements OnInit {
     /**
      * Compact mode
      */
-    compact = true;
+    compact = false;

```

Comment:
Is this some left over. You dont need this one, and you are not going to save anything using `contentDensity === 'compact'` in the template vs saving it to extra property.

Reply:
added it in template removed 'compact' property.